common/api/imodeljs-common.api.md

```diff
@@ -3012,7 +3012,7 @@ export interface GeoCoordinatesRequestProps {
     // (undocumented)
     iModelCoords: XYZProps[];
     // (undocumented)
-    targetDatum: string;
+    target: string;

```

Comment:
The api does not fit with class. Probably need to re-run extract-api

Reply:
Already pushed.